servicepulse/install-servicepulse-in-iis.md

```diff
@@ -144,61 +106,25 @@ Installation steps:
                     <match url="^api/(.*)" />
                     <action type="Rewrite" url="http://localhost:33333/api/{R:1}" />
-                </rule>
-                <rule name="Legacy rewrite main instance API URL" stopProcessing="true">
-                    <match url="^a/api(.*)" />
-                    <action type="Rewrite" url="http://localhost:33333/api/{R:1}" />
-                </rule>
+                </rule>                
             </rules>
         </rewrite>
     </system.webServer>
 </configuration>
 ```
 
-> [!WARNING]
-> When combining these rules with the ones specified above in [Detailed steps](#basic-setup-detailed-steps), ensure the rule named "Handle Vue.js routing paths" (i.e. `<match url="(.*)" />`) rule occurs _last_ in the list of rewrite rules. Otherwise, it will override the rules specified here.
-
 > [!WARNING]
 > By exposing the REST API via the reverse proxy configuration, this protection is no longer in place. To address this, it is recommended that the IIS website be configured with one of the IIS authentication providers, such as Windows integration authentication.
 
 It is also recommended that the IIS website be configured to use TLS if an authorization provider is used.
 
-#### Configuring SignalR rewrite rules
-
-Due to a [bug in SignalR](https://github.com/SignalR/SignalR/issues/3649) in Microsoft.AspNet.SignalR.JS version 2.2.0, usage of IIS as a reverse proxy requires an additional URL Rewrite Rule. This rule should look as follows:
-
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<configuration>
-    <system.webServer>
-        <rewrite>
-            <outboundRules>
-                <rule name="Update URL property" preCondition="JSON" enabled="true" stopProcessing="true">
-                    <match filterByTags="None" pattern="\&quot;Url\&quot;:\&quot;(.+?)\&quot;" />
-                    <conditions>
-                        <add input="{URL}" pattern="(.*)/api/" />
-                    </conditions>
-                    <action type="Rewrite" value="&quot;Url&quot;:&quot;{C:1}{R:1}&quot;" />
-                </rule>
-                <preConditions>
-                    <preCondition name="JSON">
-                        <add input="{URL}" pattern="/api/messagestream/negotiate" />
-                        <add input="{RESPONSE_CONTENT_TYPE}" pattern="application/json" />
-                    </preCondition>
-                </preConditions>
-            </outboundRules>
-        </rewrite>
-    </system.webServer>
-</configuration>
-```
-
 ### ServiceControl monitoring
 
 When using [monitoring capabilities](/monitoring) the following steps should be followed to create a reverse proxy to access the monitoring API from IIS.
 
 Installation steps:
 
- 1. Install the IIS [Application Request Routing extension](https://www.iis.net/downloads/microsoft/application-request-routing).
+ 1. Install the IIS [URL Rewrite Module](https://www.iis.net/downloads/microsoft/url-rewrite).
```

Comment:
A reference to the `Application Request Routing` (ARR) extension is being replaced with `URL Rewrite` Module because the latter should be enough to configure a URL rewrite. Installing ARR depends on the URL Rewrite module, which explains why installing ARR made the URL rewrites work.